resources/views/v2/styles/variables.css

```diff
@@ -61,7 +61,7 @@ $gray-dark-hover: hsl(204, 6%, 30%);
 /* Fonts */
 
 $font-family: "Sailec";
-$font-family-code: monospace;
+$font-family-code: "Cousine";
```

Comment:
monospace to "Cousine", typo?

Categories: sans-serif, serif, cursive, monospace, fantasy

I believe that we are using monospace and sans-serif only.